src/functional/fencil_processors/runners/gtfn_cpu.py

```diff
@@ -0,0 +1,61 @@
+# GT4Py Project - GridTools Framework
+#
+# Copyright (c) 2014-2022, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from typing import Any, Sequence
+
+import numpy
+
+from functional.fencil_processors import source_modules
+from functional.fencil_processors.builders import cpp as cpp_callable
+from functional.fencil_processors.codegens.gtfn import gtfn_module as gtfn_codegen
+from functional.iterator import ir
+from functional.iterator.processor_interface import fencil_executor
+
+
+def get_arg_types(
+    *args,
```

Comment:
I'm not a huge fan of defining a wrapper function for a very simple and readable list comprehension: it introduces overheads and the readability and encapsulation enhancements are small in my opinion, but it's ok for now. But I don't see the point at all of using variadic signature instead of a single argument function receiving an iterable.
```suggestion
    args,
```